android/src/main/java/com/dooboolab/flutterinapppurchase/AndroidInappPurchasePlugin.java

```diff
@@ -435,6 +437,57 @@ public void onAcknowledgePurchaseResponse(BillingResult billingResult) {
       });
     }
 
+    else if(call.method.equals("PriceChangeConfirmation")){
```

Comment:
I hope the method name to starts with the lowercase so it looks like `priceChangeConfirmation`.